pkg/nodeps/values.go

```diff
@@ -77,7 +77,7 @@ var ValidWebserverTypes = map[string]bool{
 	WebserverNginxGunicorn: true,
 }
 
-var ValidNodeJSVersions = []string{"14", "16", "18"}
+var ValidNodeJSVersions = []string{"14", "16", "18", "19", "20"}
```

Comment:
Note, 14 is still valid until 2023-04-30 as per https://github.com/nodejs/Release

Reply:
Should we drop 14 then, as this will be in DDEV v1.22, after 2023-04-30? I think probably so?

Reply:
```suggestion
var ValidNodeJSVersions = []string{ "16", "18", "19", "20"}
```

Reply:
I feel mixed about it, is the policy to always keep this up to date with supported NodeJS versions? Our own internal project didn't switch from 14 to 16 until Sep 29, 2022, which wasn't that long ago. 14 is still valid to download, but just isn't getting any security updates. I would lean towards leaving it as a possible option.

Reply:
Should we leave out the odd number (non-LTS) releases like `19`?

Reply:
If the idea is "supported", then 19 is supported for now, true?

I'm also mixed, but I'm mixed about this whole thing because we have `ddev nvm` and don't need to even have the package-installed technique :( So maybe we should remove this whole technique and just `ddev nvm install 18` by default.

Also, nodesource packages fail to download have the time, leading to false fails on tests.

Reply:
19 is only supported through the end of May, that's it.

Reply:
I would be okay with only LTS versions being listed here and removing 19. We enjoy using this configuration to enforce a standard version of nodejs across 10+ sites with a shared configuration file.

Reply:
I dont' think that will meet people's expectations, since people are always using ancient nodejs for ages. They don't know what's lts and what's not...

Reply:
If people are already using ancient nodejs, that's their problem and they should be using `ddev nvm`? I think there's an interest here in just always defaulting to the NodeJS LTS.